Editor/AssetBundleManageTab.cs

```diff
@@ -156,6 +178,100 @@ internal void OnGUI(Rect pos)
                     m_HorizontalSplitterRect.x,
                     m_VerticalSplitterRectLeft.y - m_Position.y);
                 
+                Rect searchRect = bundleTreeRect;
+                searchRect.height = 20;
+                bundleTreeRect.y += 20;
+                bundleTreeRect.height -= 20;
+
+                string uiSearchString = "";
+                SearchMode uiSearchMode = m_SearchMode;
+                
+                // using reflection to access internal -> ToolbarSearchField(Rect position, string[] searchModes, ref int searchMode, string text)
+                MethodInfo[] infos = typeof(EditorGUI).GetMethods( BindingFlags.NonPublic | BindingFlags.Static );
```

Comment:
We should expose the ToolbarSearchField if we really want to use it for this. Until it is exposed, this code should probably all live in a separate class with an identical API so it is easy to replace / refactor.